standards/mobile_app_standards.md

```diff
@@ -0,0 +1,135 @@
+Mobile Application Standards
+============================
+
+Scope
+-----
+
+These standards apply to mobile application in-house development and management.
+
+These standards are intended to cover all forms of mobile application: from fully "off-line capable" progressive web applications, through cross-platform apps to fully native apps.
+
+As implied, there are a number of ways of delivering mobile apps. These standards will cover which approach is best and under what circumstances, and aspects that are common to all or many of the approaches. As and when more detail is available on specific approaches, such as Flutter app development, or Progressive Web Apps, such approach-specific standards should form part of their own separate, detailed, documented standard, which, ideally, will simply be a reference to a recognised external standard for that specific technology.
+
+Caveats
+-------
+
+The huge increase in the use of mobile devices means that mobile application development is now of critical interest to the DEFRA family, even more so given the amount of field-work being carried out by workers within the DEFRA family.
+
+This is a fast moving area involving multiple platforms and diverse technology stacks: these standards are therefore early and provisional. As such, these standards often require more background explanation and justification than those for other, more mature or more homogenous, technology stacks. Inevitably, therefore, these standards, for the time being, are likely to be more discursive than those for other areas, and, unavoidably, the line between standard and guidance can often, as yet, not be so sharply drawn.
+
+This document should therefore be read in conjunction with the Mobile Application Guidance.
+
+Which Mobile Platforms Must Be Supported
+----------------------------------------
+
+Mobile applications must run on mobile devices. Currently DEFRA has adopted Apple technology as the standard platform for mobile devices. **All mobile applications intended for internal use must therefore run on Apple devices. All mobile applications intended for use beyond the DEFRA family, by any form of "extended enterprise" or the general public, must run on at least Apple and Android devices.**
+
+Working Offline
+---------------
+
+Users of mobile devices, and especially DEFRA family field workers who will often be working in remote, rural locations, must be able to continue to use any mobile applications even when lack of signal prohibits any form of internet access. In fact, **continuing to be readily usable without an internet connection, including the ability to work with data locally on the device, must be considered a defining characteristic of a truly mobile app.**
```

Comment:
Being able to work offline is probably/possibly the driving requirement for an _internal_ app but not necessarily an external one. That could be driven by a need for market presence on an app store or engagement with particular communities.

Reply:
Working offline is my definition of what it means for an app to be usefully defined as a mobile app. I think the offline capability is the critical differentiator that makes an app "mobile" and that any other definition would be unclear and potentially dangerous.

Reply:
I've now reviewed the common definitions of "mobile apps" online, and have revisited this, using instead the term "offline mobile app" to both attempt to avoid the dangerous confusion where-by projects often want mobile apps because of a need to work offline but haven't made that fully explicit even in their own thinking, and to stress that usually we will be providing offline mobile apps (but agree that usually isn't necessarily always).